UM/Settings/InstanceContainer.py

```diff
@@ -67,9 +69,8 @@ def __hash__(self):
         return id(self)
 
     def __deepcopy__(self, memo):
-        new_container = self.__class__(self._id)
-        new_container._name = self._name
-        new_container._definition = self._definition
+        new_container = self.__class__(self.getId())
+        new_container._name = self.getName()
```

Comment:
I think this isn't used anymore?

Reply:
You're right. I removed it here: 894c9d581cf7ac6bc250408e53ca448cf91e04fa